library/core/class.authenticator.php

```diff
@@ -184,7 +184,7 @@ public function fetchData($dataSource, $directSupplied = []) {
 
                     case self::DATA_COOKIE:
                         $this->_DataHooks[$dataTarget]['value'] = $this->_DataSource->getValueFrom(
-                            Gdn_Authenticator::INPUT_COOKIES,
+                            Gdn_Request::INPUT_COOKIES,
```

Comment:
I am not sure if this is correct, but `Gdn_Authenticator::INPUT_COOKIES` does not exist.